functions/Install-DbaInstance.ps1

```diff
@@ -290,7 +290,19 @@ function Install-DbaInstance {
                 $output += "[$key]"
                 if ($Content.$key -is [hashtable]) {
                     foreach ($sectionKey in $Content.$key.Keys) {
-                        $output += "$sectionKey=`"$($Content.$key.$sectionKey -join ',')`""
+                        $origVal = $Content.$key.$sectionKey
+                        if ($origVal -is [array]) {
+                            $output += "$sectionKey=`"$($origVal -join ',')`""
+                        } else {
+                            if ($origVal -is [int]) {
+                                $origVal = "$origVal"
+                            }
+                            if ($origVal -ne $origVal.Trim('"')) {
+                                $output += "$sectionKey=`"$($origVal.Trim('"') -join ',')`""
```

Comment:
Since this can no longer be an array, no need to `-join` it here. And since this is the section where the value is already quoted without the join perturbations it's just $origVal.
```suggestion
                                $output += "$sectionKey=$origVal"
```

Reply:
agreed, same thing, cleaner code